participant/tests/test_api.py

```diff
@@ -268,15 +268,17 @@ def test_participant_scores_debrief_behaviour_responses(self):
         ]
 
         for act, behaviour in test_data:
-            response = self.behaviour_response(act)
-            self.assertEqual(status.HTTP_200_OK, response.status_code)
-            self.assertEqual(1, len(response.data["phishing_emails"]))
-            self.assertEqual(
-                1, len(response.data["phishing_emails"][0]["participant_actions"])
-            )
-            self.assertEqual(
-                behaviour, response.data["phishing_emails"][0]["participant_behaviour"]
-            )
+            with self.subTest(act):
```

Comment:
could you change this to `with self.subTest(act=act, behaviour=behaviour):` so it's within scope of the `with`

Reply:
I did that but I used `act` only to mark label or description. As far as I believe it still maintains the scope.